### PR TITLE
Add daemonset commands

### PR DESCRIPTION
**Pull request by eyusupov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,16 @@ $ fisher install blackjid/plugin-kubectl
 | kdpvc        | `kubectl describe pvc`                               |
 | kdelpvc      | `kubectl delete pvc`                                 |
 
+### Daemonset management.
+
+| Abbreviation | Command                                              |
+| ------------ | ---------------------------------------------------- |
+| kgds         | `kubectl get pvc`                                    |
```
Comment (blackjid) [inline README.md]: can you fix the readme so the command matches the real aliased command?

**Review by blackjid — CHANGES_REQUESTED** [review]
Thanks! I left a small comment

File: README.md (hunk 1 above)
Comment (eyusupov) [reply README.md]: Updated

**Review by blackjid — APPROVED** [review]

**Comment by blackjid** [comment]
Thanks @eyusupov

**Pull request merged**